src/fluent-plugin-mdsd/gemspec-template

```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+    s.name = "fluent-plugin-mdsd"
+    s.version = "GEMVERSION"
+    s.date = "2017-01-05"
+    s.summary = "Fluentd output plugin for Linux monitoring agent (mdsd)"
+    s.description = "Fluentd output plugin for Linux monitoring agent(mdsd), which sends data to Azure Geneva service."
+    s.authors = ["azlinux"]
+    s.email = "azlinux@microsoft.com"
```

Comment:
change to my alias for now before there is a team-wise alias.

---
In reply to: [114165351](https://github.com/Azure/fluentd-plugin-mdsd/pull/26#discussion_r114165351) [](ancestors = 114165351)